packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -363,6 +364,17 @@ class Tooltip extends StatefulWidget {
   /// If this property is null, [MouseCursor.defer] will be used.
   final MouseCursor? mouseCursor;
 
+  /// Whether this tooltip should be visible to hit testing.
```

Comment:
Nit: "visible" => "invisible"

Just because `true` means it's invisible, not visible.